_scripts/collect_contributor_data.R

```diff
@@ -28,13 +28,18 @@ excluded_handles <- c(
 )
 # A vector of handles of contributors who contributed outside of GitHub
 included_handles <- c(
-  "Na-Nino",
-  "dgmascarina" # linelist logo and graphical summary
+  "CatalinaGU",
+  "dgmascarina", # linelist logo and graphical summary
+  "hawyndiaz",
+  "mauricio110785",
+  "modiazv",
+  "Na-Nino"
 )
 
 repos <- lapply(repos, function(x) {
   if (!x$name %in% excluded_repos) {
-    one_repo <- get_contributors(org = org_name, repo = x$name)
+    one_repo <- get_contributors(org = org_name, repo = x$name, check_urls = FALSE)
+    Sys.sleep(5)
```

Comment:
```suggestion
    # It seems we are hitting GitHub API rate limit. We circumvent this issue with Sys.sleep() for now
    # but keep an eye on https://github.com/ropenscilabs/allcontributors/issues/XX for a potentially
    # better solution.
    Sys.sleep(5)
```

Could you open an issue for this please? IIRC, GitHub sends a specific error code or message for rate limit so it should be possible to:
- at least give a warning
- potentially retry when rate-limited